cmd/jaeger/config.yaml

```diff
@@ -32,6 +32,7 @@ extensions:
       traces_archive: another_store
     ui:
       config_file: ./cmd/jaeger/config-ui.json
+    max_clock_skew_adjust: 5s
```

Comment:
we don't want this in the default config

Reply:
Sure no problem, I'll remove it again.

Was just wondering where to put this to make people aware. Because the v2 docs are a WIP with only sample configs.

Even had to clone the repo and search through the code for a good 20 minutes or so to even realize it exists 😅.

So any other example config where I should put it or does it warrant a new config?

Reply:
you can add it commented out, the the same value as the current default.